package.json

```diff
@@ -36,8 +36,9 @@
   },
   "homepage": "https://github.com/azu/nlp-pattern-match",
   "devDependencies": {
+    "@types/node": "14.6.2",
```

Comment:
I recommened that just update typescript or use `resolusions` field.

- [Selective dependency resolutions | Yarn](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/)